spyderlib/config.py

```diff
@@ -713,7 +712,7 @@ def is_ubuntu():
 # 2. If you want to *remove* options that are no longer needed in our codebase,
 #    you need to do a MAJOR update in version, e.g. from 3.0.0 to 4.0.0
 # 3. You don't need to touch this value if you're just adding a new option
-CONF_VERSION = '16.1.0'
+CONF_VERSION = '17.1.0'

```

Comment:
Yes. @d1manson, you need to do rebase your PR because another PR changed this value.

The right final value (according to the numbers present here) should be `17.0.0`, not `17.1.0`